tests/Driver/API/ExceptionConverterTest.php

```diff
@@ -29,6 +29,8 @@ protected function setUp(): void
     /**
      * @param class-string<DriverException> $expectedClass
      *
+     * @psalm-suppress RedundantCondition see: https://github.com/vimeo/psalm/issues/4473
```

Comment:
We usually use `psalm.xml` for this. I think the reason is to have them all in a centralized place.

Reply:
Also, I think you should link to the PHPUnit issue

Reply:
Done!